### DISCUSSION TITLE
Open app while click button from email

### DISCUSSION THREAD

**Discussion by shariful-selise** [nklayman/vue-cli-plugin-electron-builder#1765] [General]
I am building a electron app with vue cli and vue cli plugin builder.. 
I don't know how to open electron app when some one click from a button link from email.
If someone know this part how to do it please help me.

**Comment by Barrosy** [comment]
What have you tried so far to achieve this goal? 

From what I understand is that usually opening apps from the browser it is not recommended to do so as this might [impose security risks](https://stackoverflow.com/a/2800089/3437688). 

Nevertheless, in programming world nothing is impossible (as far as my knowledge reaches that is). For Electron apps, you might find a solution to your question. I found a post on Stack Overflow where someone tried to explain the possibility of executing an Electron app from the browser [here](https://stackoverflow.com/a/53603876/3437688). Perhaps this might suit your needs?
